docs/release-notes/2.43.0.md

```diff
@@ -0,0 +1,27 @@
+# Percona Monitoring and Management 2.43.0
+
+| **Release date** | July , 2024                                                                                     |
+| -----------------| ----------------------------------------------------------------------------------------------- |
+| **Installation** | [Installing Percona Monitoring and Management](../quickstart/index.md) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+It enables you to observe the health of your database systems, explore new patterns in their behavior, troubleshoot them and execute database management operations regardless of whether your databases are located on-premises or in the cloud.
+
+## What's new in this release
+
+This release introduces this and that
+
+## Release highlights
+
+### Feature name
+
+![!image](../_images/.png)
+
+## Improvements
+
+- [PMM-13133](https://perconadev.atlassian.net/browse/PMM-13133) - Following our usual maintenance routine, we upgraded the package `pg_query_go`, which helps parse PostgreSQL queries in QAN, to the latest version v5. This will slightly degrade the functionality of Query Analytics, which will not be able to list PostgreSQL database tables for versions 9.4, 9.5, and 9.6.
+
```

Comment:
@catalinaadam 
Can we add something like: "Reminder the PG 9 is EOL and you. should get the new version, you can install the latest version of PErcona Distor.  for PG [Link to our doc ]"

Reply:
## Maintenance

### QAN update: Upgrade to `pg_query_go` v5 and impact on PostgreSQL 9.x

As part of our ongoing maintenance efforts, we have upgraded the `pg_query_go` package to version 5. This latest version is essential for parsing PostgreSQL queries in Query Analytics (QAN), providing enhanced functionality, and enabling us to upgrade other important dependencies.

However, with this change, QAN will no longer be able to list PostgreSQL database tables for versions 9.4, 9.5, and 9.6. Since all PostgreSQL 9.x versions have already reached their end-of-life status, we recommend upgrading to a supported version of PostgreSQL. 

If you're looking to upgrade, you can easily [install the latest version of Percona Distribution for PostgreSQL](https://docs.percona.com/postgresql/13/installing.html)].

Reply:
@rnovikovP, Would this cover it? If we don't have other maintenance updates, I could rephrase the title  to "QAN maintenance update: Upgrade to pg_query_go v5 and impact on PostgreSQL 9.x". WDYT?